tests/e2e/contracts/football_prediction_market.py

```
@@ -0,0 +1,97 @@
+# { "Depends": "py-genlayer:test" }
+
+from genlayer import *
```

Flake8 (7.2.0)

[error] 'from genlayer import *' used; unable to detect undefined names

(F403)